src/Components/ArtworkAlert/Components/Steps/Details.tsx

```diff
@@ -0,0 +1,114 @@
+import ChevronRightIcon from "@artsy/icons/ChevronRightIcon"
+import {
+  Box,
+  Button,
+  Checkbox,
+  Clickable,
+  Flex,
+  Join,
+  Separator,
+  Spacer,
+  Text,
+} from "@artsy/palette"
+import { Formik } from "formik"
+import { FC } from "react"
+
+import { ArtworkAlertNameInputQueryRenderer } from "Components/ArtworkAlert/Components/Form/ArtworkAlertNameInput"
+import { CriteriaPillsQueryRenderer } from "Components/ArtworkAlert/Components/CriteriaPills"
+import { DetailsInput } from "Components/SavedSearchAlert/Components/DetailsInput"
+import { PriceRangeFilter } from "Components/ArtworkAlert/Components/Form/PriceRange"
+import { useArtworkAlertContext } from "Components/ArtworkAlert/Hooks/useArtworkAlertContext"
+import { useFeatureFlag } from "System/useFeatureFlag"
+
+export interface ArtworkAlertFormikValues {
+  name: string
+  push: boolean
+  email: boolean
+  details: string
+}
+
+export const Details: FC = () => {
```

Comment:
**walkthrough**: Similar to the form fields in the current flow. I've simplified the content a bit based on features that we don't plan to release soon (like "Hear from an Artsy Art Advisor" and "frequency"). We can always add them in later when we're ready.